src/PrestaShopBundle/Resources/views/Admin/AdvancedParameters/system_information.html.twig

```diff
@@ -0,0 +1,280 @@
+{#**
+ * 2007-2017 PrestaShop
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2017 PrestaShop SA
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ *#}
+{% extends 'PrestaShopBundle:Admin:layout.html.twig' %}
+
+{% block content %}
+{% if notHostMode %}
+    <script type="text/javascript">
+        $(document).ready(function()
+        {
+            $.ajax({
+                type: 'POST',
+                url: '{{ path("admin_system_information_check_files") }}',
+                data: {},
+                dataType: 'json',
+                success: function(json)
+                {
+                    var tab = {
+                        'missing': '{{ "Missing files"|trans({}, "Admin.Advparameters.Notification") }}',
```

Comment:
If you add this at the top of the template:
```twig
{% trans_default_domain "Admin.Advparameters.Notification" %}
```
then you won't need to pass parameters to `|trans`

Reply:
I've used a regex to convert smarty translation calls to twig ones. Less noise is always better => done!